docs/custom_text_ops.md

```diff
@@ -0,0 +1,365 @@
+## Operator Schemas
+
+### Auxiliary String Operator
+
+|**Operator**|**Support State**|
+|------------|-----------------|
+|StringEqual |  Supported        |
+|StringHash  |  Supported        |
+|StringToHashBucketFast|Supported|
+|StringJoin  | Supported         |
+|StringRegexReplace| Supported  |
+|StringSplit | Supported       |
+|StringUpper  | Supported     |
+|StringSlice | In next version |
+|StringLength | In next version |
+|StringToVector|  In next version|
+|VectorToString| In next version |
+
+
+
+### Tokenizer
+
+|**Operator**|**Support State**|
+|------------|-----------------|
+|GPT2Tokenizer| Supported       |
+|BertTokenizer| In next version |
+|XLNetTokenizer| In next version |
+
+
+## Auxiliary String Operator
+
+[TODO: Add existing operators]
+
+### <a name="StringSlice"></a><a name="StringSlice">**StringSlice**</a>
+Do the slice operation to each string element in input tensor. Similar to string slice in python
+```python
+a = "abcdef"
+b = a[1:2]
+c = a[3:1:-1]
+```
+#### Inputs
+
+***data: tensor(string)***
+<dd>String tensor to extract slices from.</dd>
+
+***starts: tensor(int64/int32)***
+<dd>The tensor of starting indices of corresponding string in data, which has same dimension of data.</dd>
+
+***ends: tensor(int64/int32)***
+<dd>The tensor of ending indices of corresponding string in data, which has same dimension of data.</dd>
+
+***steps(optional): tensor(int64/int32)***
```

Comment:
Due to ORT API limit, the optional input cannot be supported. Alternatively, we could assign a default value for the node.